src/collections_and_containers/python/linked_list/src/linked_list.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Python Version: 3.12
+"""
+File: linked_list.py
+Author: Vlad421
+Date: 1/19/2024
+
+Description: Linked list collection implementation
+"""
+
+
+class LinkedList:
+
+    """Linked list data structure representation
+    """
+
+    class Node:
+        def __init__(self, value: [any]) -> None:
+            self.value = value
+            self.next = None
+
+    def __init__(self) -> None:
+        """Creates empty linked list"""
+        self.__data: list[self.Node] = list()
+        self.__number_of_elements: int = 0
+
+    def empty(self) -> bool:
+        """Returns True if empty, False if not empty"""
+        return self.__number_of_elements == 0
+
+    def size(self) -> int:
+        """Returns True if empty, False if not empty"""
+        return self.__number_of_elements
+
+    def push_back(self, value: any) -> None:
+        """Add element to end of list"""
+        element = self.Node(value)
+        if not self.empty():
+            self.__data[-1].next = element
+
+        self.__data.append(element)
+        self.__number_of_elements += 1
+
+    def front(self) -> any:
+        """Returns first element"""
+        element: any = 0
+        if not self.empty():
+            element = self.__data[0].value
+        else:
+            element = None
+        return element
```

Comment:
can be done just without `element` with returns

Reply:
It's done on purpose. Turns out everything in Python is object. Wasn't sure if the return statement returns value or reference.